libs/ui-lib/lib/common/config/constants.ts

```diff
@@ -372,7 +372,7 @@ export const CUSTOM_MANIFESTS_HELP_LINK =
 
 //TODO (mortegag): Change the link. Related to https://issues.redhat.com/browse/MGMT-13976
```

Comment:
Looks like the TODO should be removed now.

Reply:
True. I've changed it